internal/shim/kubectl.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2022 Outreach Corporation. All Rights Reserved.
+
+package shim
+
+import (
+	_ "embed"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+)
+
+//go:embed kubectl.embed
+var kubectlScript []byte
+
+// AddKubectl creates a kubectl shell script in $HOME/.local/dev-environment/bin. This script executes
+//
+//		devenv --skip-update kubectl "$@"
+//
+// The function also adds the script to $PATH so it's picked by child processes.
+func AddKubectl(opts ...func(*Options)) error {
```

Comment:
```suggestion
func AddKubectl(opts ...func(*Options)) error {
```

nit: Create a `Option` type